pkg/config/settings.go

```diff
@@ -66,12 +82,23 @@ func LoadDefaultsOfIBMBlockCSI() error {
 	if err != nil {
 		return fmt.Errorf("error unmarshaling yaml: %v", err)
 	}
+	return nil
+}
 
+func setDefaultSidecarImageByName() {
 	DefaultSidecarsByName = make(map[string]v1.CSISidecar)
 
 	for _, sidecar := range DefaultCr.Spec.Sidecars {
 		DefaultSidecarsByName[sidecar.Name] = sidecar
 	}
+}
 
-	return nil
+func setDefaultControllerImageByName() {
+	DefaultControllerByName = make(map[string]v1.IBMBlockCSIControllerSpec)
+	DefaultControllerByName["ibm-block-csi-controller"] = DefaultCr.Spec.Controller
 }
+
+func setDefaultNodeImageByName() {
+	DefaultNodeByName = make(map[string]v1.IBMBlockCSINodeSpec)
+	DefaultNodeByName["ibm-block-csi-node"] = DefaultCr.Spec.Node
+}
```

Comment:
I think you missed this while going over your diff

Reply:
missed what? the `ibm-block-csi-node` string?